main.go

```diff
@@ -170,6 +178,17 @@ func run(cmd *cobra.Command, _ []string) {
 		managerOptions.Namespace = watchNamespace
 	}
 
+	if observabilityServiceEndpoint != "" {
+		otel.SetLogger(logger)
+		tp, err := trace.CreateTraceProvider(observabilityServiceEndpoint, version)
+		if err != nil {
+			logger.Error(err, "unable to create traceprovider")
+			os.Exit(1)
+		}
+		otel.SetTracerProvider(tp)
+		otel.SetTextMapPropagator(otel_propagation.NewCompositeTextMapPropagator(otel_propagation.TraceContext{}, otel_propagation.Baggage{}))
```

Comment:
Nice! This way we can support propagators in W3C Trace Context format (https://www.w3.org/TR/trace-context/) and W3C Baggage format out of the box.